### DISCUSSION TITLE
Mapping data

### DISCUSSION THREAD

**Discussion by pascalprey** [firefly-iii/firefly-iii#5107] [Firefly III] [answered]
Hey guys,

at first thank you so much for developing firefly-iii. I really like it, it is amazing!

I have a question: I would like to define some mappings.
E.g. "AMAZON PAYMENTS EUROPE S.C.A." to "Amazon"

I took a look on the documentation of firefly-iii and found the correct page. 
But my problem is that I can't find the correct page in my firefly-iii instance where I have to map the entries. 

Can you help me?

Regards
Pascal

**Comment by JC5** [comment, marked as answer]
Mapping data is something that you do in the CSV importer, not in Firefly III itself. Is that what you're looking for?

**Reply by pascalprey** [reply]
Thank you! I tried to map the description data but I think this is not possible. 

My bank has no description column in export data, so I am using the opposing bank account as description in my firefly instance.

**Reply by edwardjamesgaff** [reply]
> Mapping data is something that you do in the CSV importer, not in Firefly III itself. Is that what you're looking for?

Hopefully I'm not committing a faux pas by reviving this question but I'm having the same confusion. I just started trying out Firefly III and I'm playing with the CSV Data Importer.

I now understand that the mapping data feature happens as part of the Importer but when I select a column to map I'm never taken to a screen to map it.

Re-reading the documentation, it seems that mapping can only be done with certain fields? I've been setting the Description field to the transaction recipient (e.g. Amazon) column. With the help of this thread I think I've pieced the breadcrumbs together in this way:

1. The Description field cannot be mapped.
2. It may not be best practice to set the transaction recipient to the Description field

Are these assumptions true? And perhaps more importantly, is the mapping feature supposed to be able to be used in the way that the OP (and I) were expecting: e.g. mapping multiple Amazon entries to an "Amazon" group (for lack of a better word)?

I played with setting the transaction recipient to Opposing Account but it expects an account defined in Firefly III to map it to. I highly doubt that I should be creating accounts for every merchant. This makes me think I'm attempting to use mapping in a way it wasn't intended to be used.

**Reply by JC5** [reply]
First of all: it's not necessary to revive old questions. You can just as easily open a new discussion and refer to the old one. There's plenty of space on the internet.

But to answer your question, both things are true.

> I played with setting the transaction recipient to Opposing Account but it expects an account defined in Firefly III to map it to. I highly doubt that I should be creating accounts for every merchant. This makes me think I'm attempting to use mapping in a way it wasn't intended to be used.

If you map data, yes. So create one first. If you don't map data, accounts will be created for you.
